third_party/gpus/cuda/platform.bzl

```diff
@@ -0,0 +1,39 @@
+CUDA_VERSION = ""
+CUDNN_VERSION = ""
+PLATFORM = ""
+
+def cuda_sdk_version():
+  return CUDA_VERSION
+
+def cudnn_sdk_version():
+  return CUDNN_VERSION
+
+def cuda_library_path(name, version = cuda_sdk_version()):
+  if PLATFORM == "Darwin":
+    return "lib/lib{}.{}.dylib".format(name, version)
+  else:
+    return "lib64/lib{}.so.{}".format(name, version)
```

Comment:
Looks like you were viewing an outdated diff - this was fixed in a previous commit: https://github.com/tensorflow/tensorflow/pull/664/commits/0933043bf4691c72ae3af95d5df3b7e197a3516f